corporate/tests/stripe_fixtures/attach_discount_to_realm--Charge.create.1.json

```diff
@@ -39,7 +39,7 @@
     "network_status": "approved_by_network",
     "reason": null,
     "risk_level": "normal",
-    "risk_score": 00,
+    "risk_score": 0,
```

Comment:
@hackerkid you understand how these fixtures were generated from the Stripe API -- does Stripe send values with this incorrect normalization?  I guess we can try running the tool to regenerate these fixtures with this `orjson` branch running and see if re-introduces them and/or fails?

Reply:
More verification is always appreciated, but I’m assuming the incorrect normalization came from our code, specifically this line which I adjusted in the same commit.

```diff
@@ -181,8 +181,8 @@ def normalize_fixture_data(decorated_function: CallableT,
                 if match not in normalized_values[pattern]:
                     normalized_values[pattern][match] = translation % (len(normalized_values[pattern]) + 1,)
                 file_content = file_content.replace(match, normalized_values[pattern][match])
-        file_content = re.sub(r'(?<="risk_score": )(\d+)', '00', file_content)
-        file_content = re.sub(r'(?<="times_redeemed": )(\d+)', '00', file_content)
+        file_content = re.sub(r'(?<="risk_score": )(\d+)', '0', file_content)
+        file_content = re.sub(r'(?<="times_redeemed": )(\d+)', '0', file_content)
         file_content = re.sub(r'(?<="idempotency-key": )"([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f-]*)"',
                               '"00000000-0000-0000-0000-000000000000"', file_content)
         # Dates

```

Reply:
@timabbott Stripe doesn't return back `00`. The code above replaces the `risk_score` returned by Stripe with `00` before saving the fixture. The reason I went with 00 instead of 0 at that time was that the values of `risk_score` and `times_redeemed` returned by Stripe and in turn present in fixtures were both 2 digit numbers. (`risk_score` actually can have any value between 0 and 100 and `times_redemmed` could be any positive integer). So the decision to go with 00 instead of 0 was mostly to mimic how the values looked like in those fixtures.

So the change made by Anders is correct. We will now store 0 instead of 00 in test fixtures. And it doesn't break anything.